src/plugins/gnome-software-install-appstream.in

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
```

Comment:
Doesn't really matter, but there's nothing `bash` specific here. Could just be `#!/bin/sh -e`.